Update README how to deploy theme (#8)
vshn/keycloak-theme: README.md

# keycloak-theme
VSHN Keycloak IDP Theme

## Local Development

Run `docker-compose up` to start a local keycloak with the vshn theme.

## Deploy to Your Instance

> 1. Merge your Pull Request with the theme to master

Reviewer: ```suggestion
1. Merge your Pull Request to master branch
```

1. Tag it with
    ```bash

>     git tag -am v1.x.y

Reviewer: those flags are probably not needed?
```suggestion
    git tag v1.x.y
```

Author: ah, I wanted to write

```
git tag -am v1.x.y v1.x.y
```

I once was told that I should tag with `--annotate` and `--message` and I do it since then. Do you think it's ok with an unannotated tag?

Reviewer: For the release it doesn't matter. But you can keep your -am flags, no hard feelings here :)

    git push --tags
    ```

> 1. This will trigger an image push on https://quay.io/repository/vshn/keycloak-theme?tab=tags

Reviewer 2: ```suggestion
1. This will trigger the GitHub Action and pushes the image to https://quay.io/repository/vshn/keycloak-theme?tab=tags
```

> 1. This will trigger a deployment on https://quay.io/repository/vshn/keycloak-theme?tab=tags
> 1. Add the tag of the version to your extraInitContainers ([see codecentric helm chart documentation](https://github.com/codecentric/helm-charts/blob/master/charts/keycloak/README.md#providing-a-custom-theme))
>     ```yaml
>     extraInitContainers: |
>       - name: theme-provider
>         image: quay.io/vshn/keycloak-theme:v1.0.0
>     ```

Reviewer: ```suggestion
1. This will trigger an image push on https://quay.io/repository/vshn/keycloak-theme?tab=tags
1. Add the tag of the version to your extraInitContainers ([see codecentric helm chart documentation](https://github.com/codecentric/helm-charts/blob/master/charts/keycloak/README.md#providing-a-custom-theme))
    ```yaml
    extraInitContainers: |
      - name: theme-provider
        image: quay.io/vshn/keycloak-theme:v1.x.x
    ```
```
